Sources/Core/AST/Stmt/ConditionalCompilationStmt.swift

```diff
@@ -94,10 +94,70 @@ public struct ConditionalCompilationStmt: Stmt {
 
   }
 
+  /// A sequence of structured conditions to elaborate a 1 to N operand compilation condition
+  public indirect enum SequenceCondition: Codable {
+    case operand(Condition)
+    case and(Condition, SequenceCondition)
+    case or(Condition, SequenceCondition)
```

Comment:
```suggestion
  public indirect enum SequenceCondition: Codable {

    case operand(Condition)

    case and(Condition, SequenceCondition)

    case or(Condition, SequenceCondition)
```

We add spaces after/before the opening/closing braces of types and we add spaces between member declarations. The same rule should be applied to the lines below.